### PR TITLE
[WIP]Update all doc after M5 regenesis on stage2

### PR DESCRIPTION
**Pull request by AnastasiiaVashchuk**

### CODE REVIEW

File: docs/api/js/accounts-l1-l2.md [hunk 1]
```diff
@@ -179,27 +180,37 @@ async claimFailedDeposit(depositHash: BytesLike): Promise<ethers.ContractTransac
...
+| transaction.operatorTip (optional) | If the ETH `value` passed with the transaction is not explicitly stated in the overrides, this field will be equal to the tip the operator will receive on top of the base cost of the transaction. This value has no meaning for the `Deque` type of queue, but it will be used to prioritize the transactions that get into the `Heap` or `HeapBuffer` queues. 
+                                                                                                        |
+| overrides (optional)               | Ethereum transaction overrides. May be used to pass `gasLimit`, `gasPrice` etc.
+                                |
+| transaction.gasPerPubdataByte (optional)      |
+                              | 
+| transaction.refundRecipient(optional)         |  
+|--                                                                                                                                                         
+| returns                               | `PriorityOpResponse` object.
```
Comment (AnastasiiaVashchuk) [inline docs/api/js/accounts-l1-l2.md:213]: @krofax  I updated the context of the table, but it is displayed very bed in the preview for me...could you please correct it

File: docs/api/js/accounts-l1-l2.md [hunk 2]
```diff
@@ -228,20 +239,21 @@ const abi = [
...
 });
 
 console.log(`Executing the transaction will cost ${ethers.utils.formatEther(txCostPrice)} ETH`);
 
 const executeTx = await wallet.requestExecute({
-  calldata,
-  l2gasLimit,
   contractAddress: "0x19a5bfcbe15f98aa073b9f81b58466521479df8d",
+  calldata,
+  l2Value: 1,
```
Comment (StanislavBreadless) [inline docs/api/js/accounts-l1-l2.md:255]: why do we supply 1 here?

File: docs/dev/developer-guides/bridging/l1-l2.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ This section describes the interface for interaction with zkSync from L1. It ass
 
 ::: warning
 
-Please note that in the new `0.13.0-beta.2` SDK we changed the API to rename ergs to gas, to avoid breaking changes in your integration, update all your project dependencies.
+Please note that in the new `0.13.0-beta.2` SDK the API layer operates with gas. The ergs consept is used by VM only.
```
Comment (StanislavBreadless) [inline docs/dev/developer-guides/bridging/l1-l2.md:10]: Please change it in all other places too

```suggestion
Please note that in the new `0.13.0-beta.2` SDK the API layer operates with gas. The ergs concept is used by VM only.
```

Comment (uF4No) [reply docs/dev/developer-guides/bridging/l1-l2.md]: fixed

File: docs/api/js/accounts-l1-l2.md (hunk 1 above)
Comment (uF4No) [reply docs/api/js/accounts-l1-l2.md:212]: fixed

File: docs/api/js/accounts-l1-l2.md (hunk 2 above)
Comment (krofax) [reply docs/api/js/accounts-l1-l2.md:254]: @AnastasiiaVashchuk any reason why?

**Review by krofax — CHANGES_REQUESTED** [review]

File: docs/api/js/accounts-l1-l2.md [hunk 4]
```diff
@@ -73,17 +73,17 @@ async deposit(transaction: {
...
-| transaction.amount                      | The amount of the token to be deposited.                                                                                                                                                                                                                                                                                                                         |
-| transaction.to (optional)               | The address that will receive the deposited tokens on L2.                                                                                                                                                                                                                                                                                                        |                                       
+| Name                                    | Description      |
+| --------------------------------------- | ------- |
+| transaction.token                       | The address of the token to deposit  |
```
Comment (arose0) [inline docs/api/js/accounts-l1-l2.md:78]: nitpick: missing period after "deposit".

File: docs/api/js/accounts-l1-l2.md [hunk 5]
```diff
@@ -153,7 +153,7 @@ async getBaseCost(params: {
 
 | Name                        | Description                                                                                            |
 | --------------------------- | ------------------------------------------------------------------------------------------------------ |
-| params.l2gasLimit           | The `l2gasLimit` for the call.                                                                          |
+| params.gasLimit             | The `gasLimit` for the call.                                                                          |
```
Comment (arose0) [inline docs/api/js/accounts-l1-l2.md:156]: Just a suggestion, but maybe it is worth making it clear when we're talking about L1 and L2 here. I note below that we still say "the gas price of the L1 transaction" and we should make sure we're being as unambiguous as possible here.

File: docs/api/js/accounts-l1-l2.md (hunk 2 above)
Comment (AnastasiiaVashchuk) [reply docs/api/js/accounts-l1-l2.md:254]: @StanislavBreadless  randomly, or I understand the mission of this variable wrong?

Comment (AnastasiiaVashchuk) [reply docs/api/js/accounts-l1-l2.md:254]: @StanislavBreadless  there is also contract address value... For me looks random - there is no explanation for what is it

Comment (StanislavBreadless) [reply docs/api/js/accounts-l1-l2.md:254]: @AnastasiiaVashchuk `l2Value` is a field which denotes how much ETH to you want to be present within this L1->L2 transaction

Comment (AnastasiiaVashchuk) [reply docs/api/js/accounts-l1-l2.md:254]: so if it is an example why we can not just pass `l2Value` = 1?

File: docs/api/js/accounts-l1-l2.md (hunk 5 above)
Comment (AnastasiiaVashchuk) [reply docs/api/js/accounts-l1-l2.md:156]: absolutely agree!
will fix.

File: docs/api/js/accounts-l1-l2.md (hunk 2 above)
Comment (StanislavBreadless) [reply docs/api/js/accounts-l1-l2.md:248]: @AnastasiiaVashchuk we can, but then we should explain what we are doing here. It is just in the previous example no value was passed

**Review by krofax — APPROVED** [review]

**Pull request merged**